configuration/mounting-paritions/en.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Automatically mounting internal partitions"
++++
```

Comment:
Oh, yea, GitHub didn't differentiate between the Git addition mark and usual + in TOML lol.